rust/worker/src/execution/operators/merge_knn_results.rs

```diff
@@ -95,7 +96,25 @@ impl Operator<MergeKnnResultsOperatorInput, MergeKnnResultsOperatorOutput>
                     if input.include_vectors {
                         hnsw_result_vectors = Some(Vec::new());
                     }
-
+                    // Prefetch data.
+                    // Note: This will be moved to an operator.
```

Comment:
Will we be moving the prefetch out of his operator or be subdispatching?

Reply:
discussed offline. Removed this code and will do the right thing